docs/CoC.md

```diff
@@ -0,0 +1,211 @@
+  the community organizers.
+* A permanent ban with right of appeal.
+* A permanent ban without right of appeal, by unanimous consent of the community
+  organizers.
+* Temporary revocation of community organizer status for any length of time, with 
+  right of appeal.
+* Temporary revocation of community organizer status for any length of time,
+  without right of appeal, by unanimous consent of the remaining community
+  organizers.
+* Permanent revocation of community organizer status with right of appeal.
+* Permanent revocation of community organizer status without right of appeal, by
+  unanimous consent of the remaining community organizers.
+* Removal of contributions to Pext from any Pext-associated repositories, with
+  right of appeal.
+* Removal of contributions to Pext from any Pext-associated repositories,
+  without right of appeal, by unanimous consent
+* Notification of law enforcement, or other appropriate authorities, along with
+  any evidence of wrongdoing provided to the community organizers.
+
+## Weapons policy
+
+No weapons are allowed at Pext community events, community spaces, or in other
+spaces covered by the scope of this Code of Conduct. Weapons include, but are
+not limited to, guns, explosives (including fireworks without prior approval by
+a community organizer), and large knives such as those used for hunting or
+display, as well as any other item used for the purpose of causing injury and
+harm to others. Anyone seen in the possession of these items will be asked to
+leave immediately, and will only be allowed to return without the weapon.
+Community members are further required to comply with all local laws and
+regulations on this matter.
+
+Repeated violations of this policy are grounds for enforcement actions as if
+against unacceptable behaviour.
+
+## Reporting guidelines
+
+If you are subject to, or witness, unacceptable behaviour, or have any other
+concerns, please notify a community organizer as soon as possible. The current
+list of community organizers is:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+* Koz Ross <koz.ross@retro-freedom.nz>
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engages in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc)
+* Who (besides yourself) was affected by this behaviour.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Additionally, community organizers are available to help community members
+engage with local law enforcement or otherwise help those experiencing
+unacceptable behaviour feel safe. In the context of in-person events, organizers
+will also provide escorts as desired by the person experiencing distress.
```

Comment:
Uuuuhm. Like, I agree that this would be good! But I... don't know much about helping with local law enforcement and stuff. I think this paragraph should be removed because it's promising things that I don't know if I can keep. Sure, I'll always try to help, but I don't want to explicitly promising things I am not 100% sure I can keep.

Reply:
It basically means 'we will give any evidence we have to the LEOs if necessary, and we will advocate on your behalf'. You can't really do much more than that if you're not a lawyer, and I doubt anyone espects you to. Would a rephrase to something like that be OK?

Reply:
It's good to give evidence over to authorities but I don't think it's a good idea to become involved voluntarily in a legal case. Not all of us would be willing or able to.

Reply:
Again, poor choice of words on my part. How about I rephrase this to:

"Community organizers are able to contact, and give evidence to, local law enforcement pertaining to any unacceptable behaviours you are subject to on your behalf. We are also able to help you otherwise feel safe."

Reply:
What about: "Community organizers may assist local authorities and connect you with help or providing a safe environment."

Reply:
I'd be happy with that, except I'd use 'can' instead of 'may'.

Reply:
Sounds good to me.